lua/graph.lua

```diff
@@ -1658,7 +1658,8 @@ function filter_tags_generic(kv)
      kv["link_type"] = kv["link_type"]
   end
 
-  kv["private"] = private[kv["access"]] or private[kv["motor_vehicle"]] or "false"
+  kv["private"] = private[kv["access"]] or private[kv["motor_vehicle"]] or private[kv["motorcar"]] or "false"
```

Comment:
OMG.. and as I wrote it down it dawned on me.. I added `private[kv["motorcar"]]` (which makes sense IMO). reverting that lets the isochrones succeed.

Reply:
bike is affected by normal destonly in that it adds a 10 mins penalty btw.

Reply:
I also verified it in the PBF: there's some `motorcar=destination` just around the place where the geometry was impacted. Instead of reverting that change, I updated the isochrone geometries.